sdc/tests/test_series.py

```diff
@@ -293,7 +294,7 @@ def test_impl():
 
     def test_create_series_index2(self):
```

Comment:
We might add docstrings for these tests to clarify what they are testing and what's the difference between them.

Reply:
added